### PR TITLE
Added chapter about automation-bundle

### PR DESCRIPTION
**Pull request by wachterjohannes**
| Q | A
| --- | ---
| Fixed tickets | none
| License | MIT

#### What's in this PR?

This PR add the documentation for the feature branch https://github.com/sulu/sulu/pull/3037

### CODE REVIEW

**Comment by wachterjohannes** [comment]
@danrot @webmozart @chirimoya what do you think? should we include some screenshots in here?

File: bundles/automation/handler.rst [hunk 1]
```diff
@@ -0,0 +1,98 @@
+Custom Handler
+==============
+
+Custom handler provides functionality which can be scheduled with the
```
Comment (danrot) [inline bundles/automation/handler.rst]: `A custom handler`

File: bundles/automation/index.rst [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+specific timestamp. This Handlers (like publish) can be implemented by the
+application or any bundle.
+
+.. figure:: ../../img/automation-tab.png
+	:align: center
+
+Basic Ideas
+-----------
+
+The basic idea behind this bundle was to enhance the content-manager with the
```
Comment (danrot) [inline bundles/automation/index.rst]: content manager

File: bundles/automation/index.rst [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+possibility to plan when a page will be online. But also other tasks should
+be possible. Envisaged scenario could be to send an E-Mail after 6 months to
+check the content of the page and many other scenarios.
```
Comment (danrot) [inline bundles/automation/index.rst]: `One of many scenarios could be to send a mail after 6 months to check the content of the page.` And I somehow doubt that this is a good example...

File: bundles/index.rst [hunk 4]
```diff
@@ -14,17 +14,18 @@ documented the most important ones.
...
+    automation/index
     collaboration
     content/index
     hash
     http_cache
+    markup/index
     media
     persistence
+    preview/index
+    route/index
```
Comment (danrot) [inline bundles/index.rst]: Were all these pages missing?

Comment (wachterjohannes) [inline bundles/index.rst]: no i have sorted it by alphabet (:

**Comment by wachterjohannes** [comment]
@danrot fixed

**Pull request merged**